### DISCUSSION TITLE
Introduce myself

### DISCUSSION THREAD

**Discussion by linox2000** [community/community#20084] [Other Feature Feedback, Questions, & Ideas]
Hello I'm **ManuelLino** from Angola, fascinated by the tech world especially web developer with react and NodeJS. I always look for opportunities to evolve and never stop learning.

**Comment by aleemaheer** [comment]
Good to see you here ManuelLino, greetings from Pakistan.

**Reply by linox2000** [reply]
thanks

**Comment by Haeya** [comment]
Hello I'm Haeya from Korea, Working as a web developer with NestJS and Django.
